documentation/docs/extensibility.md

```diff
@@ -1,86 +1,184 @@
+For example, you can't reorder stages, change which stages run in parallel or add new stages.
 
-## Further tips and information
+You can copy and paste the existing Pipelines from those sources into your `Jenkinsfile` and make changes to it
 
-When you consider to add additional capabilities your first stop should be the [Jenkins Pipeline Steps Reference](https://jenkins.io/doc/pipeline/steps/).
-Here you get an overview about what kind of capabilities are already available and a list of related parameters which you can use to customize the existing implementation. The provided information should help you to understand and extend the functionality of your pipeline.
+* [piperPipeline](https://github.com/SAP/jenkins-library/blob/master/vars/piperPipeline.groovy)
+* [SAP Cloud SDK Pipeline](https://github.com/SAP/cloud-s4-sdk-pipeline-lib/blob/master/vars/cloudSdkPipeline.groovy)
 
-!!! tip
-    If you consider extensions we recommend you to do it using a custom library according to the [Jenkins shared libraries](https://jenkins.io/doc/book/pipeline/shared-libraries/) concept instead of adding groovy coding to the `Jenkinsfile`.
-    Your custom library can easily live next to the provided pipeline library.
+### Multiple projects
 
-    Your Jenkinsfile would then start like
+Similar to what you can do in an individual `Jenkinsfile`, you can also copy the pipeline to a file in a separate git repository and modify it.
 
-    ```
-    @Library(['piper-lib-os', 'your-custom-lib']) _
+To do this, create a new git repository in your preferred git hosting service.
+It must be compliant to [how Jenkins shared libraries are built](https://jenkins.io/doc/book/pipeline/shared-libraries/).
+In a nutshell, this means you need a `vars` directory inside which you can place a copy of your preferred pipeline.
 
-    ```
+A minimal example of such a library could have this directory structure:
+
+```
+./vars/myCustomPipeline.groovy
+./README.md
+```
+
+where `myCustomPipeline.groovy` contains the modified pipeline code.
+
+!!! note
+    Your custom pipeline _must_ be named different from the other pipelines provided by project "Piper", because Jenkins requires names across multiple libraries to be unique.
+
+This library needs to be placed in a git repository which is available for Jenkins and must be configured in Jenkins [as documented here](https://jenkins.io/doc/book/pipeline/shared-libraries/#using-libraries).
+
+![Library Setup](images/customPipelineLib.png "Library Setup")
+
+The `Jenkinsfile` would look similar to this:
+
+```groovy
+@Library(['piper-lib-os@vINSERT_VERSION_HERE','my-own-pipeline@vINSERT_VERSION_HERE']) _
+
+myCustomPipeline script: this
+```
+
+Be sure to adapt the names and version identifiers accordingly.
+
+### How to stay up-to-date
+
+Regardless which of the above options you chose, one downside of this approach is that your pipeline will be out of sync with the centrally maintained pipelines at some point in time.
```

Comment:
```suggestion
Regardless which of the above options you choose, one downside of this approach is that your pipeline will be out of sync with the centrally maintained pipelines at some point in time.
```

"chose" is past tense, unless you actually meant that here. (Since this documentation is about a thing the user has likely not implemented yet, present tense seems more appropriate.)